### PR TITLE
Added vocab and shapes metadata to core and cm

### PR DESCRIPTION
**Pull request by jamsden**

### CODE REVIEW

File: specs/cm/change-mgt-shapes.ttl [hunk 1]
```diff
@@ -20,6 +20,19 @@
...
+  dcterms:description "Constraints on vocabulary terms defined in the OSLC Change Management (CM) namespace."^^rdf:XMLLiteral ;
+  dcterms:publisher <https://open-services.net/about/> ;
+  dcterms:issued "2020-08-06"^^<http://www.w3.org/2001/XMLSchema#date> ;
+  dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
+  dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/cm/change-mgt-shapes.ttl> ;
+  dcterms:isPartOf <https://oslc-op.github.io/oslc-specs/specs/cm/change-mgt-spec.html> ;
+  dcterms:hasVersion "PS01" .
 
 
 <#ChangeRequestShape> a oslc:ResourceShape ;
```
Comment (berezovskyi) [inline specs/cm/change-mgt-shapes.ttl:38]: All the resources should have the same base here.

I suggest `@prefix : <http://open-services.net/ns/cm/shapes/3.0/#> . ` and then `: a oslc:ResourceShapeConstraints` and `:ChangeRequestShape a oslc:ResourceShape`.

File: specs/cm/change-mgt-vocab.ttl [hunk 2]
```diff
@@ -28,10 +28,15 @@
 oslc_cm:
   a owl:Ontology ;
   dcterms:title "The OSLC Change Management(CM) Vocabulary" ;
+  rdfs:label "Change Management(CM)" ;
```
Comment (berezovskyi) [inline specs/cm/change-mgt-vocab.ttl:31]: As Nick suggested in my PR, let's have the same `title` and `label`.

File: specs/cm/change-mgt-vocab.ttl [hunk 3]
```diff
@@ -28,10 +28,15 @@
...
   dcterms:description "All vocabulary URIs defined in the OSLC Change Management (CM) namespace."^^rdf:XMLLiteral ;
-  dcterms:source <https://tools.oasis-open.org/version-control/svn/oslc-ccm/trunk/specs/change-mgt/change-mgt-vocab.ttl> ;
   vann:preferredNamespacePrefix "oslc_cm" ;
-  rdfs:label "Change Management(CM)" .
+  dcterms:publisher <https://open-services.net/about/> ;
+  dcterms:issued "2020-08-06"^^<http://www.w3.org/2001/XMLSchema#date> ;
```
Comment (berezovskyi) [inline specs/cm/change-mgt-vocab.ttl:35]: I don't think this is an issued file, let's make sure only to merge such a change to master once Chet approves the spec publication.

File: specs/cm/change-mgt-vocab.ttl [hunk 4]
```diff
@@ -28,10 +28,15 @@
...
+  dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
+  dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/cm/change-mgt-vocab.ttl> ;
+  dcterms:isPartOf <https://oslc-op.github.io/oslc-specs/specs/cm/change-mgt-spec.html> ;
+  dcterms:hasVersion "PS01" .
```
Comment (berezovskyi) [inline specs/cm/change-mgt-vocab.ttl:39]: same here, let's keep it WD until then. plus, we'd have to change the URIs in lockstep to point to the published file location.

File: specs/core/core-shapes.ttl [hunk 5]
```diff
@@ -21,6 +21,20 @@
 @prefix foaf:  <http://xmlns.com/foaf/0.1/> .
 @prefix oslc:  <http://open-services.net/ns/core#> .
 
+
+<http://open-services.net/ns/cm/core/3.0/>
+  a oslc:ResourceShapeConstraints ;
+  dcterms:title "The OSLC Core Constraints" ;
+  dcterms:description "Constraints on vocabulary terms defined in the OSLC Core namespace."^^rdf:XMLLiteral ;
+  rdfs:label "Core" ;
```
Comment (berezovskyi) [inline specs/core/core-shapes.ttl:29]: same here, let's have the same title and label.

File: specs/core/core-shapes.ttl [hunk 6]
```diff
@@ -21,6 +21,20 @@
...
+  dcterms:publisher <https://open-services.net/about/> ;
+  dcterms:issued "2020-08-06"^^<http://www.w3.org/2001/XMLSchema#date> ;
```
Comment (berezovskyi) [inline specs/core/core-shapes.ttl:31]: let's remove this property until the spec is approved for publishing.

File: specs/core/core-shapes.ttl [hunk 7]
```diff
@@ -21,6 +21,20 @@
...
+  dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
+  dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/core/core-shapes.ttl> ;
+  dcterms:isPartOf <https://oslc-op.github.io/oslc-specs/specs/core/oslc-core.html> ;
+  dcterms:hasVersion "PS01" .
```
Comment (berezovskyi) [inline specs/core/core-shapes.ttl:35]: let's remove this property until the spec is approved for publishing.

File: specs/core/core-vocab.ttl [hunk 8]
```diff
@@ -24,11 +24,14 @@ oslc:
   a owl:Ontology ;
   dcterms:title "The OSLC Core Vocabulary" ;
   dcterms:description "All vocabulary URIs defined in the OSLC Core namespace."^^rdf:XMLLiteral ;
-  dcterms:modified "2020-06-11"^^<http://www.w3.org/2001/XMLSchema#date> ;
+  rdfs:label "Core" ;
```
Comment (berezovskyi) [inline specs/core/core-vocab.ttl:27]: let's have the same title and label, please.

File: specs/core/core-vocab.ttl [hunk 9]
```diff
@@ -24,11 +24,14 @@ oslc:
...
+  vann:preferredNamespacePrefix "oslc" ;
   dcterms:publisher <https://open-services.net/about/> ;
+  dcterms:issued "2020-08-06"^^<http://www.w3.org/2001/XMLSchema#date> ;
```
Comment (berezovskyi) [inline specs/core/core-vocab.ttl:30]: let's remove this property until the spec is approved for publishing.

File: specs/core/core-vocab.ttl [hunk 10]
```diff
@@ -24,11 +24,14 @@ oslc:
...
+  dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
   dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/core/core-vocab.ttl> ;
-  vann:preferredNamespacePrefix "oslc" ;
-  rdfs:label "Core" .
+  dcterms:isPartOf <https://oslc-op.github.io/oslc-specs/specs/core/oslc-core.html> ;
+  dcterms:hasVersion "PS01" .
```
Comment (berezovskyi) [inline specs/core/core-vocab.ttl:34]: let's remove this property until the spec is approved for publishing.

File: specs/cm/change-mgt-shapes.ttl (hunk 1 above)
Comment (jamsden) [reply specs/cm/change-mgt-shapes.ttl:38]: The oslc-core spec does not have the # and /# seems strange. Maybe we should have said .../3.0#, not .../3.0/ for proper fragments. But I'm not sure fragments matter in a .ttl file.

File: specs/cm/change-mgt-vocab.ttl (hunk 2 above)
Comment (jamsden) [reply specs/cm/change-mgt-vocab.ttl:31]: Fixed, made all dcterms:title and rdfs:label content the same.

File: specs/cm/change-mgt-vocab.ttl (hunk 3 above)
Comment (jamsden) [reply specs/cm/change-mgt-vocab.ttl:35]: Fixed

File: specs/cm/change-mgt-vocab.ttl (hunk 4 above)
Comment (jamsden) [reply specs/cm/change-mgt-vocab.ttl:39]: Fixed.

File: specs/cm/change-mgt-shapes.ttl (hunk 1 above)
Comment (jamsden) [reply specs/cm/change-mgt-shapes.ttl:38]: ShapeChecker didn't like not having the fragment. So I took off the /, as in .../3.0#.

File: specs/core/core-vocab.ttl (hunk 10 above)
Comment (jamsden) [reply specs/core/core-vocab.ttl:34]: Fixed

**Pull request merged**